api/src/main/java/com/xwiki/azureoauth/internal/EntraIDObjectUpdateListener.java

```diff
@@ -150,4 +172,47 @@ private WikiReference getCurrentWikiReference()
     {
         return new WikiReference(this.wikiManager.getCurrentWikiId());
     }
+
+    /**
+     * Prior to XWiki 15.3, to be able to select the Authentication service by using the AuthService Backport, the
+     * default XWikiAuthServiceComponent needs to be selected. To do this, we check the selected authentication service
+     * and reset it to default one if externally set, with the exception if the authentication service is set in
+     * xwiki.cfg. To be removed when upgrading the parent to a version >= 15.3.
+     */
```

Comment:
Could you add the jira issue for when the AuthService implementation was done in platform?